### PR TITLE
feat(cb2-5819): create group 1 hgv/trl template

### PR DESCRIPTION
**Pull request by leifkemp-bjss**
## CB2-5819: Create desk based test group 1 for HGV/TRL

_Test group 1 for HGV/TRL_
[CB2-5819](https://dvsa.atlassian.net/browse/CB2-5819)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/forms/templates/test-records/section-templates/vehicle/desk-based-default-hgv-trl-vehicle-section.template.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    {
+      name: 'euVehicleCategory',
+      label: 'EU Vehicle Category',
+      value: '',
+      disabled: true,
+      type: FormNodeTypes.CONTROL,
+      width: FormNodeWidth.XXS
+    },
+    {
+      name: 'odometerReading',
```
Comment (tom-bjss) [inline src/app/forms/templates/test-records/section-templates/vehicle/desk-based-default-hgv-trl-vehicle-section.template.ts:45]: Odometer and everything below it not on ticket

Comment (tom-bjss) [reply src/app/forms/templates/test-records/section-templates/vehicle/desk-based-default-hgv-trl-vehicle-section.template.ts:45]: Ignore that they're all hidden

**Review by tom-bjss — APPROVED** [review]

**Review by kpearmanbjss — DISMISSED** [review]

File: src/app/forms/templates/test-records/section-templates/vehicle/desk-based-default-hgv-trl-vehicle-section.template.ts (hunk 1 above)
Comment (Ruairidh-BJSS) [inline src/app/forms/templates/test-records/section-templates/vehicle/desk-based-default-hgv-trl-vehicle-section.template.ts:45]: The odometer reading/units for a trailer doesn't seem right, on all other trailers it's not there and doesn't accept a value even if it's null

Comment (leifkemp-bjss) [reply src/app/forms/templates/test-records/section-templates/vehicle/desk-based-default-hgv-trl-vehicle-section.template.ts]: Split this into two templates, one with odometer for hgv, one without for trl

**Review by Ruairidh-BJSS — DISMISSED** [review]

**Review by HazelWilderspin — DISMISSED** [review]

**Review by Ruairidh-BJSS — DISMISSED** [review]

**Review by kpearmanbjss — DISMISSED** [review]

**Review by Ruairidh-BJSS — APPROVED** [review]

**Review by HazelWilderspin — APPROVED** [review]

**Pull request merged**